### PR TITLE
  book: Move /opt redirection guidance to building

### PR DESCRIPTION
**Pull request by cgwalters**
This is where I expected to find it.

### CODE REVIEW

**Review by ckyrouac — APPROVED** [review]

File: docs/src/building/guidance.md [hunk 1]
```diff
@@ -138,3 +138,34 @@ for example, although this can run afoul of SELinux labeling.
...
+- Put configuration files in `/usr` (if they're static), or `/etc` if they need to be machine-local
+- Put "data" (log files, databases, etc.) underneath `/var`
+
+However, some software installs to `/opt/<application>` and has
+all three types of data undernath its directory.  For example, it
+may write log files to `/opt/<application>/logs`.  A simple way to handle
+this is to change the directories that need to be writble to symbolic links
+to `/var`:
+
+```dockerfile
```
Comment (jeckersb) [inline docs/src/building/guidance.md:157]: small nit - inconsistent usage of `<application>` and `application`.  I probably wouldn't have even noticed this except the one is missing the closing `>` and it triggered my brain parser :laughing:

File: docs/src/filesystem.md [hunk 2]
```diff
@@ -121,18 +121,10 @@ Besides those, for other toplevel directories such as `/usr` `/opt`, they will b
...
-```
-RUN rmdir /opt/exampleapp/logs && ln -sr /var/log/exampleapp /opt/exampleapp/logs
-```
-
-Another option is to configure the systemd unit launching the service to do these mounts
-dynamically via e.g.
-```
-BindPaths=/var/log/exampleapp:/opt/exampleapp/logs
-```
+See [building images](guilding/guidance.md) for recommendations on how to build
```
Comment (jeckersb) [inline docs/src/filesystem.md:126]: broken link: `guilding` -> `building`

**Review by jeckersb — CHANGES_REQUESTED** [review]
I like this organization better too :+1: 

Broken link needs fixed.

One other nit, take it or leave it :smile:

**Comment by jeckersb** [comment]
Added #521 to get CI to catch these

File: docs/src/building/guidance.md (hunk 1 above)
Comment (cgwalters) [reply docs/src/building/guidance.md:158]: Thanks!  Good catch.

**Review by jeckersb — APPROVED** [review]

**Comment by cgwalters** [comment]
Test failures are unrelated.

**Pull request merged**